src/content/ui-api/management-api/authorization/index.md

```diff
@@ -0,0 +1,246 @@
+
+1. whether a resource is **cluster-scoped** or **namespace-scoped**
+2. if it is namespace-scoped, **in which namespace** they are to be found
+
+We'll explain next which resources are to be found in which scope or namespace.
+
+### Cluster scope
+
+In Giant Swarm management clusters in particular, two resource types are cluster-scoped, so they are not residing in any namespace:
+
+- [`Organization`]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md" >}}) defines, well, an organization
+- [`Release`]({{< relref "/ui-api/management-api/crd/releases.release.giantswarm.io.md" >}}) defines a workload cluster release to user when creating a new workload cluster, or to upgrade a workload cluster to.
+
+### Namespace `default` {#default-namespace}
+
+Next, the following resources reside in the `default` namespace:
+
+- App catalogs ([`Catalog`]({{< relref "/ui-api/management-api/crd/catalogs.application.giantswarm.io.md" >}}) and [`AppCatalogEntry`]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) resources) provided by Giant Swarm
+
+### Organization namespaces {#org-namespaces}
+
+For each [organization]({{< relref "/general/organizations" >}}) there is a namespace to be used as the standard location for storing resources. In these namespaces you will usually find:
+
+- Resources defining [workload clusters and node pools]({{< relref "/ui-api/management-api/creating-workload-clusters" >}}), unless you choose a different namespace for those
+- [Cloud provider credentials]({{< relref "/ui-api/management-api/credentials" >}}) in the form of `Secret` resources
+
+### Workload cluster namespaces {#wc-namespaces}
+
+For every workload cluster, there is a namespace with a name identical to the workload cluster name. In this namespace, resources related to apps installed in the workload cluster are located. The resources are of these types:
+
+- [`App`]({{< relref "/ui-api/management-api/crd/apps.application.giantswarm.io.md" >}}) which defines an app to be installed in a workload clusters
+- `ConfigMap` which optionally provides configuration for such an app
+- `Secret` which provides additional (confidential) configuration for such an app
+
+## Pre-defined roles {#pre-defined-roles}
+
+Giant Swarm provides some pre-defined cluster roles (`ClusterRole` resources) which allow to grant certain permissions to common sets of resources. The most important ones are:
+
+- **cluster-admin**: Grants all permissions to all resource types.
+- **read-all**: Grants read permissions (verbs: get, list, watch) to most known resource types, with the exception of `Secret` and `ConfigMap`.
+
+These cluster roles are meant to be bound to groups or individual users via role bindings in an organization namespace to set permissions for that organization only or via a cluster role binding (`ClusterRoleBinding` resource) in the cluster scope to set permissions for the entire management cluster.
```

Comment:
```suggestion
These cluster roles are meant to be bound to groups or individual users via role bindings in an organization namespace to set permissions for that organization only, or via a cluster role binding (`ClusterRoleBinding` resource) in the cluster scope to set permissions for the entire management cluster.
```

This sentence is a bit longer and contains a lot of information - I think it would be helpful to break it up a little bit to help with readability, but feel free to ignore/improve it as you see fit

Reply:
I simplified the paragraph by pointing to the examples in the typical use cases section.